Source/Surfaces/DataFileTopographyClass.f90

```diff
@@ -59,21 +60,16 @@ Module SMTopographyFromFileClass
 !     ---------
 !
       CHARACTER(LEN=STRING_CONSTANT_LENGTH), PARAMETER :: TOPOGRAPHY_FROM_FILE_KEY = "data file"
+      CHARACTER(LEN=STRING_CONSTANT_LENGTH), PARAMETER :: SIZING_KEY               = "sizing"
 !
 !     ----------------------
 !     Class type definition
 !     ----------------------
 !
       TYPE, EXTENDS(SMTopography) :: SMTopographyFromFile
```

Comment:
Separating the bi-cubic interpolation away from this topography class made it much cleaner / simpler

Reply:
Yes, OOP FTW.